src/utils/ui-utils.ts

```diff
@@ -316,4 +316,46 @@ export function convertVWToPixel(value: number) {
 
   const result = (x * value) / 100;
   return result;
+}
+
+/**
+ * Delete cookie helper function
+ * @param name name of the cookie
+ * @param path Indicates a URL path, defaults to '/'
+ * @param domain domain/host of the cookie
+ */
+export function deleteCookie(name:string, path = '/', domain?:string) {
+  if (getCookie(name)) {
+    document.cookie = name + '=' +
+      ((path) ? ';path=' + path : '') +
+      ((domain) ? ';domain=' + domain : '') +
+      ';expires=Thu, 01 Jan 1970 00:00:01 GMT';
+  }
+}
+
+/**
+ * Get cookie helper function
+ * @param name name of the cookie
+ * @returns cookie 
+ */
+export function getCookie(name:string){
```

Comment:
Add return type. You should also explain what happens when the cookie doesn't exist (returns false, undefined, or etc)